Solutions/Web Shells Threat Protection/Hunting Queries/SpringshellWebshellUsage.yaml

```diff
@@ -1,6 +1,8 @@
 id: 6911d1df-4204-43b2-a64c-3cb102551ddd
 name: Possible Webshell usage attempt related to SpringShell(CVE-2022-22965)
 description: |
+  This query searches Azure Web Application Firewall data for potential Webshell usage related to the SpringShell RCE vulnerability (CVE-2022-22965). For more information on detecting and protecting against this vulnerability, refer to Microsoft's security blog.
```

Comment:
Hi @petebryan, this query description exceeds 255 characters. Thanks

Reply:
Done